### PR TITLE
Cleaning up Indexer and Intake code, making intake smooth by matching surface speeds

### PR DESCRIPTION
**Pull request by JamisOrr**

### CODE REVIEW

File: src/main/java/frc/robot/commands/indexer/IndexerCommand.java [hunk 1]
```diff
@@ -12,11 +14,11 @@ public class IndexerCommand extends Command {
 	private final IndexerSubsystem m_indexerSubsystem;
 
 	public static IndexerCommand getFowardCommand(IndexerSubsystem indexerSubsystem) {
-		return new IndexerCommand(indexerSubsystem, -.8);
```
Comment (Gold856) [inline src/main/java/frc/robot/commands/indexer/IndexerCommand.java:15]: Forward? Reverse? Which is it?

File: src/main/java/frc/robot/commands/drive/PolarDriveCommand.java [hunk 2]
```diff
@@ -47,6 +47,7 @@ public PolarDriveCommand(DriveSubsystem subsystem, double amount, double angle)
 	public void initialize() {
 		m_target = m_driveSubsystem.getModulePositions()[0].distanceMeters + m_distance;
 		m_controller.setSetpoint(m_target);
+		m_controller.setTolerance(m_tolerance);
```
Comment (Gold856) [inline src/main/java/frc/robot/commands/drive/PolarDriveCommand.java:50]: Unrelated?

File: src/main/java/frc/robot/RobotContainer.java [hunk 3]
```diff
@@ -159,7 +158,8 @@ private void configureButtonBindings() {
 
 		// ------------------Intake Controls-----------------------------------
 		m_operatorController.button(Button.kLeftTrigger).onTrue(CommandComposer.getTeleopIntakeCommand(
-				m_intakeSubsystem, m_pneumaticsSubsystem, m_indexerSubsystem, m_arduinoSubsystem));
+				m_intakeSubsystem, m_pneumaticsSubsystem, m_indexerSubsystem,
```
Comment (Gold856) [inline src/main/java/frc/robot/RobotContainer.java]: Random change?

**Review by Gold856 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/commands/indexer/IndexerCommand.java (hunk 1 above)
Comment (JamisOrr) [reply src/main/java/frc/robot/commands/indexer/IndexerCommand.java:15]: That was because the motor should have been inverted, if you look at it I put that number to positive and inverted the motor

File: src/main/java/frc/robot/commands/drive/PolarDriveCommand.java (hunk 2 above)
Comment (JamisOrr) [reply src/main/java/frc/robot/commands/drive/PolarDriveCommand.java:50]: Yeah but it was annoying me

File: src/main/java/frc/robot/commands/indexer/IndexerCommand.java (hunk 1 above)
Comment (JamisOrr) [reply src/main/java/frc/robot/commands/indexer/IndexerCommand.java:15]: Oh wait I see what you mean lmfao

Comment (JamisOrr) [reply src/main/java/frc/robot/commands/indexer/IndexerCommand.java:15]: I'll fix that

Comment (JamisOrr) [reply src/main/java/frc/robot/commands/indexer/IndexerCommand.java:15]: Done

File: src/main/java/frc/robot/commands/indexer/IndexerCommand.java [hunk 4]
```diff
@@ -4,6 +4,8 @@
 
 package frc.robot.commands.indexer;
 
+import static frc.robot.Constants.IntakeConstants.*;
```
Comment (Gold856) [inline src/main/java/frc/robot/commands/indexer/IndexerCommand.java]: Shouldn't this be using IndexerConstants?

**Review by Gold856 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/commands/indexer/IndexerCommand.java (hunk 4 above)
Comment (JamisOrr) [reply src/main/java/frc/robot/commands/indexer/IndexerCommand.java]: Uh yes, one sec

**Review by Gold856 — APPROVED** [review]
